my-app/src/App.css

```diff
@@ -0,0 +1,112 @@
+* {
+  font-family: "Lato", sans-serif;
+  margin: 0;
+  padding: 0;
+  /* font-size: 12px; */
+}
+
+body,
+html {
+  margin: 0;
+  padding: 0;
+}
+
+.post {
+  background: #ffffff;
+  font-family: "Lato", sans-serif;
+  padding: 8px 8px;
+  margin: 0 24px;
+  color: #37474f;
+  -webkit-text-decoration: none;
+  text-decoration: none;
```

Comment:
I'm not entirely sure why those rules are needed. If they only apply to the anchor tags inside the container, I'd rather use a second selector (`.post a { ... }`) instead - I believe that would be a bit clearer.

Reply:
Yep, I see now that it makes sense to target the anchor tags inside of the container for text-decoration, not the container itself. I also think it makes more sense to just globally target `<a>` elements and apply these styles.